frontend/hub.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"bytes"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+type Client struct {
```

Comment:
it seems we don't need a Client struct right now.
just make this a `chan ...` e.g. 

     type wsWatchClient chan []byte